rotkehlchen/chain/evm/decoding/xdai_bridge/decoder.py

```diff
@@ -34,12 +34,13 @@ def __init__(
             evm_inquirer: 'EvmNodeInquirer',
             base_tools: 'BaseDecoderTools',
             msg_aggregator: 'MessagesAggregator',
-            deposit_topic: bytes,
+            deposit_topics: tuple[bytes, ...],
             withdrawal_topic: bytes | None,  # withdrawal is currently unsupported on gnosis
             bridge_address: ChecksumEvmAddress,
             bridged_asset: 'Asset',
             source_chain: ChainID,
             target_chain: ChainID,
+            peripheral_addresses: tuple[ChecksumEvmAddress, ...] | None = None,
```

Comment:
please add a docstring to explain what some of these are. For example no idea what peripheral addresses are